build.gradle

```diff
@@ -101,32 +106,18 @@ allprojects {
                 check -> warningsToPromote << check.canonicalName()
             }
 
+            warningsToPromote.removeAll([
+                'AndroidJdkLibsChecker', // ignore Android
+                'Java7ApiChecker', // tritium requires JDK8+
+                'StaticOrDefaultInterfaceMethod', // Android specific
+                'Var', // high noise, low signal
+            ])
+
             for (String checkName : warningsToPromote) {
                 options.errorprone.errorproneArgs << '-Xep:' + checkName + ':ERROR'
             }
 
             logger.info("Error-prone args: {}", options.errorprone.errorproneArgs)
-
-            if (generateErrorPronePatch) {
-                def patchChecks = [
-                    "PreferSafeLoggableExceptions",
-                    "PreferCollectionTransform",
-                    "PreferListsPartition",
-                ]
-
-                logger.info("Generating error-prone.patch for checks {} apply with:\n"
-                        + "    find . -maxdepth 3 -type f -name 'error-prone.patch' \\ \n"
-                        + "-exec bash -c \"  "
-                        + "cd \$(dirname {}) "
-                        + "&& patch -p0 -u -i error-prone.patch "
-                        + "&& rm error-prone.patch"
-                        + "\" \\;", patchChecks.join(','))
-
-                options.errorprone.errorproneArgs += [
-                    "-XepPatchLocation:" + project.projectDir.getAbsolutePath(),
-                    "-XepPatchChecks:" + patchChecks.join(',')
-                ]
-            }
```

Comment:
We could bring this back and update the check to `'true' == generateErrorPronePatch`